tempesta_fw/classifier/frang.c

```diff
@@ -531,6 +581,53 @@ enum {
 	Frang_Req_Done
 };
 
+const char *
+state_name(unsigned state)
```

Comment:
* It's used only in debug printouts, so it's better to compile it conditionally as well, at the same debug level it's used at.
* Perhaps it's better to distinguish it from regular functions somehow, like `__state_name()` or similar.
* Perhaps it's better to use a static array of strings instead of a run-time function.